lib/btrfs_test.pm

```diff
@@ -80,6 +80,16 @@ sub snapper_nodbus_restore {
     }
 }
 
+=head2 cron_mock_lastrun
+snapper-0.5 and older is using cron jobs in order to schedule and execute cleanup routines.
+Script /usr/lib/cron/run-crons looks into /etc/cron.{hourly,daily,weekly,monthly} for jobs
+to be executed. The info about last run is stored in /var/spool/cron/lastrun
```

Comment:
Even if it's obvious, you can add also:
`By updating the lastrun files timestamps, we make sure those routines won't be executed while tests are running`
or something like this, if I get it right.

Reply:
updated, resolving